src/lib/Scenes/Collection/Components/FeaturedArtists.tsx

```diff
@@ -8,24 +8,17 @@ import React from "react"
 import { TouchableHighlight } from "react-native"
 import { createFragmentContainer, graphql } from "react-relay"
 import { TrackingProp } from "react-tracking"
+import styled from "styled-components"
 
 interface FeaturedArtistsProps {
   collection: FeaturedArtists_collection
   tracking?: TrackingProp
 }
 
-interface FeaturedArtistsState {
-  showMore: boolean
-}
-
-const track: Track<FeaturedArtistsProps, FeaturedArtistsState> = _track
+const track: Track<FeaturedArtistsProps, any> = _track
 
 @track()
-export class FeaturedArtists extends React.Component<FeaturedArtistsProps, FeaturedArtistsState> {
-  state = {
-    showMore: false,
-  }
-
+export class FeaturedArtists extends React.Component<FeaturedArtistsProps, any> {
```

Comment:
Same comment about using `{}` for the state instead of `any`.
```suggestion
export class FeaturedArtists extends React.Component<FeaturedArtistsProps, {}> {
```